datalad_registry_client/opts.py

```diff
@@ -64,17 +66,23 @@ def process_args(
         if sibling not in remotes:
             raise ValueError("Unknown sibling: {}".format(sibling))
 
-    if not url:
-        if not sibling:
-            raise ValueError(
-                "Must specify URL to use when sibling isn't given")
+    urls: List[Tuple[str, str]] = []
+    if url:
+        urls.append((url, url_encode(url)))
+    elif sibling:
         url = repo.config.get("remote.{}.url".format(sibling))
-        if not url:
+        if url is None:
             raise ValueError("Could not find URL for {}".format(sibling))
+        urls.append((url, url_encode(url)))
+    else:
+        for r in repo.get_remotes(with_urls_only=True):
+            u = repo.get_remote_url(r)
+            assert u is not None
+            urls.append((u, url_encode(u)))
```

Comment:
```suggestion
            url = repo.get_remote_url(r)
            assert url is not None
            urls.append((url, url_encode(url)))
```
for consistency with previous block where it is a `url`.